fearlessTests/Common/Migration/InconsistentStateMigrationTests.swift

```diff
@@ -0,0 +1,70 @@
+import XCTest
+@testable import fearless
+import SoraKeystore
+
+class InconsistentStateMigrationTests: XCTestCase {
+
+    func testInconsistentStateCleared() throws {
+        // given
+
+        let settings = InMemorySettingsManager()
+        let keychain = InMemoryKeychain()
+
+        let chain = Chain.westend
+
+        try AccountCreationHelper.createAccountFromMnemonic(cryptoType: .sr25519,
+                                                            networkType: chain,
+                                                            keychain: keychain,
+                                                            settings: settings)
+
+        XCTAssertNotNil(settings.selectedAccount)
+
+        let migrator = InconsistentStateMigrator(
+            settings: settings,
+            keychain: InMemoryKeychain()
```

Comment:
keychain is defined at line 11

Reply:
Yes, this is done in purpose to immitate inconsistent state